### ISSUE TITLE
facebook keeps locking my account

### ISSUE THREAD

**Issue by archenemies** [Alex-Rose/fb-messenger-cli#225]
Facebook has asked me to change my password four times over the past week, could this be related to `fb-messenger-cli`, are other people experiencing the same problem, and is it new or has it happened before?

Thank you for this project by the way, it is wonderful when it works.

**Comment by adamac2** [comment]
I am experiencing the same thing, facebook seems to think I lost my password due to some phising attack. I wonder if there's a way to whitelist this program, or give it access in facebook settings somewhere through some sort of api

**Comment by wkosmos** [comment]
Same issue here, and facebook continues locking me out even after uninstalling fb-messenger-cli, which should mean no further requests are being sent right?

**Comment by nguyenngochoang** [comment]
Same here, I'm still waiting for the day someone fix this, lol.

**Comment by DanielEliasib** [comment]
Same happened here, it's kinda weird because facebook told me someone was trying to connect from Mac OS witch is not my operating system, and also the ip didn't match with my ip, is that expected behavior, or is the program trying to connect from somewhere else?

**Comment by KorusMateusz** [comment]
+1
I'm from Poland, trying to use it on my VPS in Germany(oracle cloud)and Facebook says it's trying to log in from Seattle even though https://www.iplocation.net/ does say my VPS is in Germany.

**Comment by allanlaal** [comment]
this should fake a common browser useragent to avoid getting blocked by FB

**Comment by archenemies** [comment]
> this should fake a common browser useragent to avoid getting blocked by FB

As far as I understand it, this tool is using a public API made available by Facebook - it's not pretending to be a web browser. That would be much harder and would require continuous maintenance.

**Comment by KorusMateusz** [comment]
> > this should fake a common browser useragent to avoid getting blocked by FB
> 
> As far as I understand it, this tool is using a public API made available by Facebook - it's not pretending to be a web browser. That would be much harder and would require continuous maintenance.

I don't think that Facebook would provide a public API to automate Messenger for real people - it does for apps, but in this project in package.json you can see that it uses puppeteer.

And as for the user agent - that's a good idea, I'll try to overwrite it and see if it helps.

**Comment by archenemies** [comment]
Thank you for keeping me honest.
